src/runner/provider/qti.js

```diff
@@ -143,10 +143,9 @@ var qtiItemRuntimeProvider = {
                     })
                     .catch(function(err) {
                         done(); // in case of postRendering issue, we are also done
-                        self.trigger(
-                            'warning',
-                            'Error in post rendering : ' + err instanceof Error ? err.message : err
-                        );
+                        var error = new Error('Error in post rendering : ' + err instanceof Error ? err.message : err);
```

Comment:
`var` should reflect hoisting (ESLint should have warned you)
But now you can also just use `const` instead.